src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp

```diff
@@ -0,0 +1,27 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace grmhd::ValenciaDivClean {
+/*!
+ * \brief Mutator used with `Initialization::Actions::AddSimpleTags` to
+ * initialize the `VariablesNeededFixing` to `false`
+ */
+struct AddVariablesNeededFixing {
```

Comment:
For my understanding: if this is used in initialization, why isn't it in an initialization namespace? if it is resetting a variable already in the databox, why is it named `Add...`?

Reply:
- I'm not sure, I can put it in an `Initialization` namespace :) I just parked it somewhere that wasn't "obvious horrendous" :shrug:
- it "adds" this in the sense of that the `Initialization::Actions::AddSimpleTags<AddVariablesNeededFixing>` has `Actions::SetupDataBox` add the simple tag and then `AddSimpleTags` sets it to the initial value by calling the mutator. Any suggestions on names or documentation are certainly welcome :)

Reply:
I would name this `SetVariablesNeededFixingToFalse` as that is what this mutator does 🤷

Reply:
I'm happy to do that. @fmahebert could you give feedback on the name before I do a rename? :)

Reply:
Yes, I agree with this suggestion